charts/kube-state/templates/kube-state-deployment.yaml

```diff
@@ -44,6 +47,9 @@ spec:
         args:
       {{- if .Values.global.singleNamespace }}
         - --namespaces={{ .Release.Namespace }}
+      {{- end }}
+      {{- if $namespacePools }}
```

Comment:
Can we make this an `else if` ? It redefines `--namespaces` so we want to ensure it's exclusive.

Reply:
i think i can merge the singlenamespace and this into one

Reply:
or (and .Values.global.features.namespacePools.enabled .Values.global.features.namespacePools.namespaces.names)  .Values.global.singleNamespace }}